src/intents.cs

```diff
@@ -1767,6 +1767,8 @@ public enum INMediaDestinationType : long {
 	public enum INMediaReference : long {
 		Unknown = 0,
 		CurrentlyPlaying,
+		[Watch (7,3), TV (14,5), iOS (14,5)]
```

Comment:
^ @rachelkang pointed out to me that the SDK shipping with Xcode 12.5 is for watchOS 7.4 (not 7.3)

even if Xcode 12.4 shipped the SDK for 7.2 😢 

I need to check my own PR too... I likely made that error too

Reply:
@rachelkang / @spouliot Nice catch!! Fixed, thanks!